src/lib/edge-functions/headers.mjs

```diff
@@ -5,10 +5,13 @@ export const headers = {
   DeployID: 'x-nf-deploy-id',
   FeatureFlags: 'x-nf-feature-flags',
   ForwardedHost: 'x-forwarded-host',
+  ForwardedProtocol: 'x-forwarded-proto',
```

Comment:
Is this one still needed?

Reply:
We're still using `x-forwarded-proto` in the code. `x-forwarded-host` could be removed, if we wanted - but I think it's also fine to leave it in, so it's easier to discover the headers that ef-bootstrap takes in the future 🤷